src/providers/SubscriptionProvider/SubscriptionProvider.actions.js

```diff
@@ -29,3 +34,66 @@ export function updateSubscription(payload) {
     payload
   };
 }
+export function updateSubscriptionError(payload) {
+  return {
+    type: UPDATE_SUBSCRIPTION_ERROR,
+    payload
+  };
+}
+export function comprobeSubscription(payload) {
+  return async (dispatch, getState) => {
+    const { expiryDate, androidSubscriptionState } = getState().subscription;
+    if (expiryDate) {
+      const expiryDateFormat = new Date(expiryDate);
+      const expiryDateMillis = expiryDateFormat.getTime();
+      const nowInMillis = Date.now();
+      const isExpired = nowInMillis > expiryDateMillis;
+
+      const daysGracePeriod = 3;
```

Comment:
Leave a comment explaining that this value should be modified before deploying on production